### ISSUE TITLE
"ember-data@2.14.0-beta.1" is missing "broccoli-stew" dependency

### ISSUE THREAD

**Issue by Turbo87** [emberjs/data#4957]
https://github.com/emberjs/data/blob/v2.14.0-beta.1/index.js#L8 is using `broccoli-stew` but is not declaring it as a dependency (https://github.com/emberjs/data/blob/v2.14.0-beta.1/package.json#L22)

```
Cannot find module 'broccoli-stew'
Error: Cannot find module 'broccoli-stew'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/private/tmp/my-cool-test-project/node_modules/ember-data/index.js:8:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by runspired** [comment]
@Turbo87 that file isn't using stew, it's just importing it. Got forgotten :'( I've PR'd removal.

**Comment by Turbo87** [comment]
cool, thanks!

**Comment by bmac** [comment]
Fixed in [v2.14.0-beta.2](https://github.com/emberjs/data/releases/tag/v2.14.0-beta.2).
